portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/logging/CompanyLogging.testcase

```diff
@@ -11,6 +11,8 @@ definition {
 
 	setUp {
 		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
```

Comment:
Its fine this time, but this should probably be seperate into a different commit since the intention isnt the same as the commit message itself